acceptance/setup/common/005_SyncTime.rb

```diff
@@ -4,4 +4,8 @@
 # Note: QENG-3641 would allow beaker to take care of this for us
 test_name 'Ensure hosts have synchronized clocks'
 
-timesync(hosts, {:logger => logger})
+# QENG-3786 - OSX is already running ntpd and manually running ntpdate will return 1
+applicable_hosts = hosts.select{|host| host['platform'] !~ /osx/}
+if applicable_hosts then
```

Comment:
What's this if statement trying to guard against?  Array#select will aways return an Array, and even the empty array is true.